### PR TITLE
[wip] aluon/forge pilot

### PR DESCRIPTION
**Pull request by aluon**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: testsuite/forge-pilot/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    #[clap(short, long, group = "genesis")]
+    docker_image: Option<String>,
+}
+
+fn main() {
+    let args = Args::parse();
+    let config = if let Some(path) = args.config_path {
+        ForgeConfig::read_from_file(&path)
+    } else if let Some(path) = args.s3_path {
+        ForgeConfig::read_from_s3(&path)
```
Comment (perryjrandall) [inline testsuite/forge-pilot/src/main.rs:33]: Lets just completely remove s3

Comment (aluon) [reply testsuite/forge-pilot/src/main.rs:33]: I just have this here for testing. We can migrate this over to GCS

**Comment by perryjrandall** [comment]
Discussed offline, but two things

1) Since we're going to depend on the larger aptos codebase, it makes sense to keep this as the same cli aptos-forge-cli
2) We can use the "base" cli image if its built for execution unless some heuristic like "changed forge" is true then we have to wait for the cli to build before scheduling
3) Please update the PR description whenu get a chance to describe the approach

File: testsuite/forge-pilot/tests/forge-wrapper-config.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        "all_tests": { "land_blocking": { "name": "land_blocking" } },
+        "enabled_tests": { "land_blocking": { "name": "land_blocking" } }
+      },
+      "land_blocking": {
+        "name": "land_blocking",
+        "all_tests": { "land_blocking": { "name": "land_blocking" } },
+        "enabled_tests": { "land_blocking": { "name": "land_blocking" } }
+      }
+    },
+    "default_helm_values": {
```
Comment (rustielin) [inline testsuite/forge-pilot/tests/forge-wrapper-config.json:37]: these are acutally not used anymore, feel free to remove

**Pull request closed without merging**